packages/grafana-ui/.storybook/main.ts

```diff
@@ -15,7 +15,7 @@ const stories = ['../src/Intro.mdx', storyGlob];
 // We limit icon paths to only the available icons so publishing
 // doesn't require uploading 1000s of unused assets.
 const iconPaths = Object.keys(availableIconsIndex)
-  .filter((iconName) => !iconName.includes('fa'))
+  .filter((iconName) => !iconName.startsWith('fa '))
```

Comment:
I can see the icons now, but will this still block the icons we want to avoid? 🤔

Reply:
yes, the one that wants to be avoided is `fa fa-spinner`

Reply:
Yes, that's right. I was thinking it was `fa-spinner`, without the previous `fa` 😅